src/common/queries/account.ts

```diff
@@ -35,16 +35,53 @@ export const getInstalledModules = async ({
   })
 
   const responseBody = await response.json()
-
-  if (response.ok) {
-    return responseBody.data.SmartAccount_ModuleQuery.map(
-      (module: any) => module.module,
-    )
+  const modules = responseBody.data.SmartAccount_ModuleQuery.map(
+    (module: any) => module.moduleAddress,
+  )
+  // @Review: response.ok is still true even if:
```

Comment:
did you already test that this is the case?

Reply:
yea, added a test, lmk if the chain used is supported but I think it's not